content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Migrate Kubernetes Objects Using Storage Version Migration
+
+reviewers:
+  - deads2k
+  - jpbetz
+  - enj
+  - nilekhc
+
+content_type: task
+min-kubernetes-server-version: v1.30
+weight: 60
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.30" state="alpha" >}}
```

Comment:
Improvement for `{{< feature-state for_k8s_version="v1.30" state="alpha" >}}`, consider using the new `feature_gate_name` option, making the feature state more data-driven.

```suggestion
{{< feature-state feature_gate_name="StorageVersionMigration" >}}
```

Reply:
Thanks for the suggestion @dipesh-rawat. I updated it. PTAL.